docs/src/digitising-process.md

```diff
@@ -1,184 +1,349 @@
+3. The third iteration was done using the `Helmert Transformation` again but with all Residual pixels for the
+   Ground Control Points (CPs) being under 10. 9 GCPS were used for the referencing. There were too many discrepancies
+   between the reference dataset and the georeferenced image, so it was disregarded as a viable image.
 
-7. The seventh iteration was done using a Thin Plate Spline transformation. All residual pixels were zero but this was likely a false result. This iteration had the best results for lining up the georeferenced image with the reference dataset. 102 GCPs were used to help correct discrepancies from previous iterations. The issue with this transformation was the significant warping of the polygons in the georeferenced image.
+4. The fourth iteration was done using a `Polynomial 1 Transformation` (with all residual pixels less than 10) and
+   12 GCPs. Again, there were too many discrepancies between the reference dataset and the georeferenced image but
+   there was minimal warping on the polygons.
 
-8. For the last iteration, it was decided that the Polynomial 1 transformation warped the internal polygons of Mozambique's Southern Region the least but more GCPs would help with the discrepancies between the reference dataset and the tif. 55 GCPs, with residual pixels lower than 10, were used for the georeferencing and ended up with the best result. The image below shows the georeferencer with the GCPs on the map and the GCP table showing the residuals being below 10 (Ground Control Point 40 has the highest Residual Pixels at 8.976885).
+5. The fifth iteration was done using a `Polynomial 2 Transformation` (with all residual pixels less than 10) and 17
+   GCPs. There were fewer discrepancies between the reference dataset and the georeferenced image than in the previous
+   iterations but there was slight warping on the polygons.
+
+6. The sixth iteration was done using a `Polynomial 3 Transformation` (with all residual pixels less than 10) and 18
+   GCPs. There were fewer discrepancies between the reference dataset and the georeferenced image than in the previous
+   iterations but there was warping on the polygons.
+
+7. The seventh iteration was done using a `Thin Plate Spline transformation`. All residual pixels were zero but this was
+   likely a false result. This iteration had the best results for lining up the georeferenced image with the reference
+   dataset. 102 GCPs were used to help correct discrepancies from previous iterations. The issue with this
+   transformation
+   was the significant warping of the polygons in the georeferenced image.
+
+##### Best Candidate Parameters to Use
+
+For the last iteration, it was decided that the `Polynomial 1 transformation` warped the internal polygons of
```

Comment:
Can we not use in the last iteration. Something like "After testing other transformation settings we settled on `Polynomial 1 transformation` because ....

Reply:
![image](https://user-images.githubusercontent.com/100694811/191523893-0430448b-1de9-41de-aa1a-7bc6561a0e2b.png)